Makefile

```diff
@@ -272,6 +283,12 @@ ${GOLANGCI_LINT}:
 		BINDIR=${BUILD_BIN_PATH} && \
 	curl -sfL $$URL/$$VERSION/install.sh | sh -s $$VERSION
 
+${SHELLCHECK}:
+	mkdir -p ${BUILD_BIN_PATH} && \
+	VERSION=v0.7.0 \
+	URL=https://github.com/koalaman/shellcheck/releases/download/$$VERSION/shellcheck-$$VERSION.linux.x86_64.tar.xz && \
+	curl -sfL $$URL | tar xfJ - -C ${BUILD_BIN_PATH} --strip 1 shellcheck-$$VERSION/shellcheck
```

Comment:
For security reasons, I would add sha256 check. However, not a blocker.

Reply:
I would have to hard code that or do you see any other resource for the SHA?

Reply:
Added the hardcoded sha